android/src/main/java/io/mosip/tuvali/transfer/Assembler.kt

```diff
@@ -37,9 +38,9 @@ class Assembler(private val totalSize: Int, private val mtuSize: Int = DEFAULT_C
       return seqNumberInMeta
     }
     lastReadSeqNumber = seqNumberInMeta
-    System.arraycopy(chunkData, chunkMetaSize, data, seqNumberInMeta * effectivePayloadSize, (chunkData.size-chunkMetaSize))
-    chunkReceivedMarker[seqNumberInMeta] = chunkReceivedMarkerByte
-    Log.d(logTag, "adding chunk complete at index(0-based): ${seqNumberInMeta}, received chunkSize: ${chunkData.size}")
+    System.arraycopy(chunkData, chunkMetaSize, data, seqIndexInMeta * effectivePayloadSize, (chunkData.size-chunkMetaSize))
```

Comment:
Proper formatting of the line is required

Reply:
added empty lines between the Act and Assert part of the code. Now the readability is good. Let me know if needed some changes.